R/timebased.R

```diff
@@ -85,7 +87,8 @@ timebased <- function(data_analyze, date_variable, time_unit="auto", outdir) {
     }else{
       resVars=c(resVars,varName)
 
-      if(var$type %in% c('Integer', 'Numeric')){
+      if(var$type %in% c('Numeric','Integer') &
+         length(unique(data_analyze[[varName]]))>nvals_num_to_cat){
```

Comment:
Here we can instead use var$qDistinct, which is the same thing, instead of length(unique(...)), that way we don't have to recalculate it, it's already done in the anomalies process which is ran above.

Reply:
I'll work on this tomorrow, it's been a holiday today in the US.  Would it make sense to use a larger number than 2 as the default for nvals_num_to_cat?? On the one hand, anything with only 3 or 4 values is probably not a reliable linear measure of a real phenomenon, so treating it as categorical is probably the best approach.  On the other hand, the parameter allows the user to set the threshold, so the default isn't really critical, and wrongly displaying numeric as a categorical variable makes trends harder to visualize.